File: spring_webflux_java/src/main/java/io/turner/springwebfluxjava/config/R2dbcConfig.java
package io.turner.springwebfluxjava.config;

import org.springframework.context.annotation.Configuration;
import org.springframework.data.r2dbc.repository.config.EnableR2dbcRepositories;

@Configuration
@EnableR2dbcRepositories(basePackages = "io.turner.data.r2dbc.repository")
public class R2dbcConfig {
}

File: spring_webflux_coroutine/src/main/kotlin/io/turner/springwebfluxcoroutine/config/R2dbcConfig.kt
package io.turner.springwebfluxcoroutine.config

import org.springframework.context.annotation.Configuration
import org.springframework.data.r2dbc.repository.config.EnableR2dbcRepositories

@Configuration
@EnableR2dbcRepositories(basePackages = ["io.turner.data.r2dbc.repository"])
class R2dbcConfig
